feat(encrypted profiles): profile encryption, finally!

SkyzohKey opened the pull request:
This pull-request add user profiles (en|de)cryption in the Wrapper, as well as in the UI.
This should also provide a way to add/change/remove password from an existing profile.

Commit:
feat(encrypted profiles): init

Commit:
feat(profile encryption): decrypt & encrypt now works!

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 0703c94dd7b88074b3f326f9889c32976fdf64d5
  * Your commit message does not contain a body

 * Commit: 50294cc2ab106401e8f0492c569de35f7d9efe1f
  * Your commit message does not contain a body

Guidelines are available at https://github.com/RicinApp/Ricin/blob/master/CONTRIBUTING.md#-git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

Commit:
feat(encrypted profiles): add UI + fix bugs
* This commit finish to implement encrypted profiles.
* It add a "simple" (for user, not in code) interface
  to add, change or remove a password. It also allow
  to create an encrypted profile without having to
  store not encrypted data on the disk.
* This commit also add some convenience functions to
  the Wrapper for dealing with encryption.
* It also add ToxES.

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 0703c94dd7b88074b3f326f9889c32976fdf64d5
  * Your commit message does not contain a body

 * Commit: 50294cc2ab106401e8f0492c569de35f7d9efe1f
  * Your commit message does not contain a body

Guidelines are available at https://github.com/RicinApp/Ricin/blob/master/CONTRIBUTING.md#-git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

Commit:
chore(libtoxencryptsave): install .pc file for ToxES

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 0703c94dd7b88074b3f326f9889c32976fdf64d5
  * Your commit message does not contain a body

 * Commit: 50294cc2ab106401e8f0492c569de35f7d9efe1f
  * Your commit message does not contain a body

 * Commit: 045609cbbd34563188a4b22e7720f8685e96d4c3
  * Your commit message does not contain a body

Guidelines are available at https://github.com/RicinApp/Ricin/blob/master/CONTRIBUTING.md#-git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

Commit:
chore(waf): update deps

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 0703c94dd7b88074b3f326f9889c32976fdf64d5
  * Your commit message does not contain a body

 * Commit: 50294cc2ab106401e8f0492c569de35f7d9efe1f
  * Your commit message does not contain a body

 * Commit: 045609cbbd34563188a4b22e7720f8685e96d4c3
  * Your commit message does not contain a body

 * Commit: 1fc5c83dfb41a40ab42b457f25717d2ca27383d9
  * Your commit message does not contain a body

Guidelines are available at https://github.com/RicinApp/Ricin/blob/master/CONTRIBUTING.md#-git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

Commit:
chore(Makefile): remove potential security issue

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 0703c94dd7b88074b3f326f9889c32976fdf64d5
  * Your commit message does not contain a body

 * Commit: 50294cc2ab106401e8f0492c569de35f7d9efe1f
  * Your commit message does not contain a body

 * Commit: 045609cbbd34563188a4b22e7720f8685e96d4c3
  * Your commit message does not contain a body

 * Commit: 1fc5c83dfb41a40ab42b457f25717d2ca27383d9
  * Your commit message does not contain a body

 * Commit: e0e4be55f2d09701cdc3dde9e16b11fbe845452d
  * Your commit message does not contain a body

Guidelines are available at https://github.com/RicinApp/Ricin/blob/master/CONTRIBUTING.md#-git-commit-guidelines

---
This message was auto-generated by https://gitcop.com
